src/resources/views/crud/fields/relationship/select.blade.php

```diff
@@ -0,0 +1,234 @@
+@php
```

Comment:
Hmmm Pedro what is this "relationship/select.blade.php" file? Is it the moved ```select2``` field? 

If so... I think I changed my mind about moving it inside the ```relationship``` folder. I'm sorry. Maybe it would be better to have it _outside_, just like before. And named the same. That way, it would:
- provide backwards compatibility
- be obvious if any changes have happened between the two files

Or is this ```select2``` + ```select2_multiple```? In that case I guess the fields are already there, and it's ok to have this inside ```relationship``` - but naming it ```select``` is a bit confusing...

Reply:
Indeed, as discussed the naming is not very good, but its a select2+select2 multiple field